resolwe_bio/processes/support_processors/bam_conversion.py

```diff
@@ -0,0 +1,152 @@
+"""Converting bam to bed and normalised bigwig files."""
+import os
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    Process,
+    SchedulingClass,
+    FileField,
+    StringField
+)
+
+
+class BamToBed(Process):
+    """Takes in a bam file and calculates a normalization factor in .bed format.
+    Done by sorting with samtools and transformed with bedtools.
+    """
+
+    slug = "bam-to-bed"
+    name = "Bam to bed"
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {"docker": {"image": "resolwebio/rnaseq:4.11.0"}},
+        "resources": {"cores": 1, "memory": 4096},
+    }
+    data_name = "Badtools bamtobed ({{alignment|sample_name}})"
+    version = "1.0.0"
+    process_type = "data:bed"
```

Comment:
Currently we have `data:bed:` and `data:bedpe` Data types defined in the resolwe-bio. The name of the output fields must match the the existing objects of the same type (or subtype). If one defines the type data:bedpe, use this process as a reference: https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/import_data/annotation_bedpe.py If one defines the `data:bed`, match the output fields to this process: https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/import_data/annotation_bed.yml